portfolio/src/main/java/com/google/sps/servlets/HardCodedStringServlet.java

```diff
@@ -9,11 +12,31 @@
 /** Servlet that returns HTML that contains a hardcoded string. */
 @WebServlet("/mystring")
 public class HardCodedStringServlet extends HttpServlet {
-
+    
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    ArrayList<String> messages = new ArrayList<String>();
```

Comment:
This can be `List<String> messages = new ArrayList<String>();` 
`ArrayList` is implementation detail, we only need to expose the most generic type.